lms/resources/lti_launch.py

```diff
@@ -144,3 +97,11 @@ def custom_canvas_api_domain(self):
         Canvas.
         """
         return self._request.parsed_params.get("custom_canvas_api_domain")
+
+    @property
+    def should_use_section_groups(self):
+        """Return True if section groups rather than a course group should be used."""
+        if not self.is_canvas:
+            return False
+
+        return self._request.feature("section_groups")
```

Comment:
The question of whether or not section groups should be used is going to have to become a lot more complicated before we can release this. For now you just have to be in Canvas and have the feature flag on, which will allow us to deploy and test intermediate versions of the sync API